packages/@okta/vuepress-theme-prose/components/TermsAndConditionsDialog.vue

```diff
@@ -0,0 +1,267 @@
+      <div class="terms-conditions-btns">
+        <div class="btn-container">
+          <input
+            type="button"
+            class="btn red-button"
+            @click="!isDisabledSocialAuth && setTaCUrlAndRedirect()"
+            :class="{ 'btn-disabled': isDisabledSocialAuth }"
+            value="continue"
+            :disabled="isDisabledSocialAuth"
+          />
+        </div>
+        <div class="btn-container mr-15">
+          <input
+            type="button"
+            class="btn social-btn"
+            value="cancel"
+            @click="closeTermsConditionsDialog()"
+          />
+        </div>
+      </div>
+    </template>
+  </DialogWindow>
+</template>
+
+<script>
+import {
+  countriesList,
+  americanStates,
+  canadaProvinces,
+  GDPR_COUNTRIES
+} from "../const/signup.const";
+
+const CANADA = "Canada";
+const USA = "United States";
+
+export default {
+  name: "TermsAndConditionsDialog",
+  components: {
+    SmartLink: () => import("./SmartLink"),
+    DialogWindow: () => import("./DialogWindow")
+  },
+  props: {
+    socialUrl: {
+      required: true,
+      type: String
+    }
+  },
+  data() {
+    return {
+      selectedCountry: "",
+      selectedRegion: "",
+      regionData: { type: "", list: [] }
+    };
+  },
+  computed: {
+    isDisabledSocialAuth() {
+      // If country not selected, disable "continue" button.
+      if (!this.selectedCountry) {
+        return true;
+      }
+      // If country Canada or USA but region not selected, disable "continue" button.
+      if (this.selectedCountry === USA || this.selectedCountry === CANADA) {
+        if (!this.selectedRegion) {
+          return true;
+        }
+      }
+    },
+    region: {
+      get() {
+        return this.regionData;
+      },
+      set(country) {
+        if (country === USA) {
+          this.regionData.list = americanStates;
+          this.regionData.type = "State";
+        } else if (country === CANADA) {
+          this.regionData.list = canadaProvinces;
+          this.regionData.type = "Province";
+        } else {
+          this.regionData.list = [];
+          this.regionData.type = "";
+        }
+      }
+    },
+    countrieList() {
+      return countriesList;
+    },
+    isGDPRCountry() {
+      if (GDPR_COUNTRIES.includes(this.selectedCountry)) {
+        return true;
+      }
+    }
+  },
+  methods: {
+    setTaCUrlAndRedirect() {
+      const acceptContactValue = this.$refs.gdprBoxEl.checked;
+      const countryName = this.selectedCountry;
+      const regionType = this.region.type.toLowerCase();
+      const regionName = this.selectedRegion;
+      const dataStamp = Date.now();
+      let redirectUrlParameters = [];
+
+      redirectUrlParameters = [
+        { name: "okta_AcceptedToS", value: acceptContactValue },
+        { name: "okta_ts_AcceptedToS", value: dataStamp },
+        { name: "country", value: encodeURI(countryName) },
+        { name: regionType, value: encodeURI(regionName) },
```

Comment:
I believe this should use `encodeURIComponent` because `encodeURI` does not encode `&` `+` and `=`.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURI#encodeuri_vs_encodeuricomponent